From 88348eef21ce91c0307afbe58f96f67f5cd29158 Mon Sep 17 00:00:00 2001
From: Kasual <mushroomhead52e@gmail.com>
Date: Tue, 17 Oct 2017 20:03:46 -0400
Subject: [PATCH 003/249] wiki: update gapps page

Makes **immediately** stand out more for those reading
Adds that The Open GApps Project is not supported (in addition to
gapps)

Change-Id: I436363bfcc2065ef47fb5839087971e63630fc89
---
 pages/gapps.md | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pages/gapps.md b/pages/gapps.md
index eef0e69..76f5b46 100644
--- a/pages/gapps.md
+++ b/pages/gapps.md
@@ -10,7 +10,7 @@ boot or run LineageOS, however many users find them beneficial to take full adva
 These apps have been packaged by a developer independent of LineageOS, and download links have been provided for your convenience only. It is possible to perform
 a "backup" of the Google apps on your device and then "restore" them, but this does take additional steps which are not covered here.
 
-The Google apps packages are **not supported** in any way by LineageOS.
+The Open GApps Project and the Google apps packages are **not supported** in any way by LineageOS.
 
 ## Downloads
 
@@ -23,11 +23,11 @@ These packages are only dependent on your OS version and architecture, which can
 
 ## Installation
 
-Google apps should be installed via recovery _immediately_ after installing LineageOS. Exact steps vary, but the process is similar to that of installing LineageOS:
+Google apps should be installed via recovery **immediately** after installing LineageOS. Exact steps vary, but the process is similar to that of installing LineageOS:
 
 {% include important.html content="If you reboot into LineageOS before installing Google apps, you must factory reset and then install them, otherwise expect crashes." %}
 
 1. Copy the Google apps zipfile to `/sdcard/`
     * Using [adb](adb_fastboot_guide.html): `adb push filename.zip /sdcard/`
 2. After installing LineageOS, choose "install zip" or "Apply update" in recovery, and navigate to the zipfile loaded earlier.
-3. Reboot back in to Android
+3. Reboot to system (i.e. LineageOS).
-- 
2.17.1

